.eslintrc.json

```diff
@@ -71,7 +71,8 @@
       "paths": [
         {"name": "react-bootstrap", "importNames": ["OverlayTrigger"], "message": "Please use OverlayTrigger from '/components/overlay_trigger' instead"}
       ]
-    }]
+    }],
+    "max-lines": ["warn", {"max": 800, "skipBlankLines": true, "skipComments": true}]
```

Comment:
(question) So, with this new rule, pipelines won't fail if 800 lines are exceeded correct? Has this 800 lines come as the outcome of not having files with more than 800 lines in place? Also, should we let people know about this change (so that 1. they don't continue e.g. disabling based on past experience and 2. try to follow the relaxed 800 lines rule (instead of starting e.g. to disable the rule to make the warnings go away).

I don't know if there is already in place a channel or way we communicate these changes (no matter how small) that affect others, so that's why I'm asking :)

Reply:
- Yes, since this is just a warning CI wont fail, if the rule is violated.
- 800 is just a number which came up in discussion here https://community.mattermost.com/core/pl/7fbbyid14fr9xdgbwirakjc4cy
- Yes definitely i will post a message in `developers` to communicate the new rule change

Reply:
we do announcements like these https://community.mattermost.com/core/pl/uzabupzspbbkprug5hskfc44gr